SPECS/libtar/libtar.spec

```diff
@@ -18,11 +18,11 @@ Patch2:         libtar-1.2.11-mem-deref.patch
 # CVE-2021-33643
 # CVE-2021-33644
 Patch3:         libtar-1.2.20-CVE-2021-33643-CVE-2021-33644.patch
-# CVE-2021-33645
-# CVE-2021-33646
-Patch4:         libtar-1.2.20-CVE-2021-33645-CVE-2021-33646.patch
-Patch5:         libtar-1.2.20-fix-resource-leaks.patch
-Patch6:         libtar-1.2.20-static-analysis.patch
+# CVE-2021-33640
```

Comment:
Should probably list CVE-2021-33645, CVE-2021-33646 as well for this patch in the comment